src/Compilers/VisualBasic/Portable/Emit/NamedTypeSymbolAdapter.vb

```diff
@@ -353,6 +353,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             End If
         End Function
 
+        Private Function ShouldIncludePrivateField(field As FieldSymbol) As Boolean
+            ' Backing fields for WithEvents are emitted with AccessedThroughPropertyAttribute
+            ' so need to be emitted even if private
+            Return TypeOf field Is SourceWithEventsBackingFieldSymbol
```

Comment:
>TypeOf field Is SourceWithEventsBackingFieldSymbol [](start = 19, length = 50)

Perhaps it will be more robust to check if associated symbol ```IsWithEventsProperty```?